types/wordpress__components/flex/index.d.ts

```diff
@@ -8,22 +8,35 @@ declare namespace Flex {
         align?: CSSProperties['alignItems'] | 'top' | 'bottom' | undefined;
 
         /**
-         * Determines the spacing in between children components. The `gap` value is a multiplier to the base value of `4`.
+         * The direction flow of the children content can be adjusted with `direction`. `column` will
+         * align children vertically and `row` will align children horizontally.
          */
-        gap?: number | undefined;
+        direction?: CSSProperties['flexDirection'] | undefined;
 
         /**
-         * Sets jusifty-content. Left and right are shorthand for flex-start and flex-end respectively, not the actual CSS value.
+         * Expands to the maximum available width (if horizontal) or height (if vertical).
          */
-        justify?: CSSProperties['justifyContent'] | 'left' | 'right' | undefined;
+        expanded?: boolean | undefined;
 
         /**
-         * Reverses the flex direction.
+         * Spacing in between each child can be adjusted by using `gap`. The value of `gap` works as
+         * a multiplier to the library's grid system (base of `4px`).
          */
-        isReversed?: boolean | undefined;
```

Comment:
`isReversed` is also still working but with a deprecation notice: https://github.com/WordPress/gutenberg/blob/6eff9a514591f3b14f667e94b745eaaa507d912a/packages/components/src/flex/flex/hook.js#L28